utils/cliutils/commandsflags.go

```diff
@@ -1390,6 +1393,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Go,
 		Usage: "[Default: false] Set to true to request audit for a Go project.` `",
 	},
+	goPublishExclusions: cli.StringFlag{
+		Name:  exclusions,
+		Usage: "[Optional] Semicolon-separated list of exclude patterns. Exclude patterns may contain the * and the ? wildcards or a regex pattern, according to the value of the 'regexp' option.` `",
```

Comment:
This description seems to be wrong. It says that we're allowing regexps. Are we really?